Neos.Cache/Classes/Backend/FileBackend.php

```diff
@@ -251,35 +251,27 @@ public function findIdentifiersByTag(string $searchedTag): array
     public function findIdentifiersByTags(array $tags): array
     {
         $entryIdentifiers = [];
-        $now = $_SERVER['REQUEST_TIME'];
```

Comment:
is there a reason to why we used that to calculate the expiry? (now your dto uses time())

Reply:
Mmm, I guess it makes it consistent for this request, why not. I will change it in the DTO.

Reply:
I mean i dont really know whats "better" :D time() feels a bit cleaner though ...

_technically_ its not clean to do this at all in the dto since its a sideeffect ... but if we dont tell anyone :D